source/development/packaging/generate-aix-package.rst

```diff
@@ -5,8 +5,9 @@
   
 .. _create-aix:
 
-AIX
-===
+=========
```

Comment:
```suggestion
```

Reply:
To keep consistency with other files, please remove overlining which is just optional. Same comment for the other occurrences.